content/blog/2021-11-10-agent.md

```diff
@@ -0,0 +1,156 @@
+
+While the official [Remote Write API specification is in review stage](https://docs.google.com/document/d/1LPhVRSFkGNSuU1fBd81ulhsCPR4hkSZyyBj1SZ8fWOM/edit), the ecosystem adopted the Remote Write protocol as the default metric export protocol. For example, Cortex, Thanos, OpenTelemetry, and cloud services like Amazon, Google, Grafana, Logz.io, etc.
+
+Prometheus project also offers the official compliance tests for its APIs, e.g. [remote-write sender compliance](https://github.com/prometheus/compliance/tree/main/remote_write_sender) for solutions that offer Remote Write client capabilities. It's an amazing way to quickly tell if you are correctly implementing this protocol.
+
+Streaming data from such scraper allows to fulfil Global View and store important information in separate locations. This also enables great separation of concerns, which is useful when applications are managed by different teams than the observability or monitoring pipelines. Furthermore, it is also why Remote Write is chosen by vendors who want to offload their customers from as much work as possible.
+
+> Wait for a second, Bartek. You just mentioned before that pushing metrics directly from the application is not the best idea!
+
+Sure, but the amazing part is that, even with the Remote Write, Prometheus still uses a pull model to gather metrics from applications, which gives us an understanding of those different failure modes. After that, we batch samples and series and export, replicate (push) data to the Remote Write endpoints, limiting the number of monitoring unknowns central point has!
+
+It's important to note that a reliable and efficient remote-writing implementation is a non-trivial problem to solve. The Prometheus community spent around three years to come with a stable and scalable implementation. We reimplemented WAL (Write-Ahead-Log) a few times, added internal queuing, sharding, smart back-offs and more. All of this is hidden from the user who can enjoy well-performing streaming or large amounts of metrics off clusters.
+
+### Hands-on Remote Write Example: Katacoda Tutorial
+
+All of this is not new in Prometheus. Many of us already use Prometheus to scrape all required metrics and remote-write all or some of them to remote locations.
+
+Suppose you would like to try the hands-on experience of remote writing capabilities. In that case, we recommend our [Thanos Katacoda tutorial of remote writing metrics from Prometheus](https://katacoda.com/thanos/courses/thanos/3-receiver), which explains all steps required for Prometheus to forward all metrics to the remote location. It's **free**, just sign up for an account and enjoy the tutorial! 🤗
```

Comment:
> "our ... tutorial"

Does this tutorial officially belong to the Prometheus project? Otherwise, I would qualify what "our" means.

Reply:
"🤗" -> I find hug emojis a bit inappropriate (creepy?) in a technical / professional context, TBH